it/info_test.py

```diff
@@ -16,18 +16,31 @@
 # under the License.
 
 import it
+from esrally.utils import process
 
 
-@it.random_rally_config
+@it.rally_in_mem
```

Comment:
I opted to switch all of these tests to the in-memory configuration because it does not matter at all which metrics store implementation we use here.